### PR TITLE
cloud: add credit docs

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)
cloud: add credit docs

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.2 (TiDB 6.2 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: TOC-tidb-cloud.md [hunk 1]
```diff
@@ -170,7 +170,7 @@
   - [Data Transfer Cost](/tidb-cloud/tidb-cloud-billing.md#data-transfer-cost)
   - [Invoices](/tidb-cloud/tidb-cloud-billing.md#invoices)
   - [Billing Details](/tidb-cloud/tidb-cloud-billing.md#billing-details)
-  - [Trial Points](/tidb-cloud/tidb-cloud-billing.md#trial-points)
+  - [Trial Points](/tidb-cloud/tidb-cloud-billing.md#credits)
```
Comment (TomShawn) [inline TOC-tidb-cloud.md:173]:
```suggestion
  - [Credits](/tidb-cloud/tidb-cloud-billing.md#credits)
```

**Review by yuezhanglang — APPROVED** [review]
LGTM

File: tidb-cloud/tidb-cloud-billing.md [hunk 2]
```diff
@@ -112,13 +112,24 @@ To view the billing details, perform the following steps:
 2. Click **Billing**.
 3. Click **Bills**. The billing details page is displayed.
 
-## Trial points
+## Credits
 
-During the [PoC](/tidb-cloud/tidb-cloud-poc.md) period, you can use trial points to pay the TiDB cluster fees. One point is equivalent to one U.S. dollar. All your TiDB clusters will be automatically terminated when the trial points are used up.
+You can use credits to pay TiDB cluster fees before the credits become expired. One credit is equivalent to one U.S. dollar.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-billing.md:117]:
```suggestion
TiDB Cloud offers a certain number of credits for Proof of Concept (PoC) users. One credit is equivalent to one U.S. dollar. You can use credits to pay TiDB cluster fees before the credits become expired.

You can use credits to pay TiDB cluster fees before the credits become expired. One credit is equivalent to one U.S. dollar.
```

File: tidb-cloud/tidb-cloud-billing.md [hunk 3]
```diff
@@ -112,13 +112,26 @@ To view the billing details, perform the following steps:
 2. Click **Billing**.
 3. Click **Bills**. The billing details page is displayed.
 
-## Trial points
+## Credits
 
-During the [PoC](/tidb-cloud/tidb-cloud-poc.md) period, you can use trial points to pay the TiDB cluster fees. One point is equivalent to one U.S. dollar. All your TiDB clusters will be automatically terminated when the trial points are used up.
+TiDB Cloud offers a certain number of credits for Proof of Concept (PoC) users. One credit is equivalent to one U.S. dollar. You can use credits to pay TiDB cluster fees before the credits become expired.
+
+You can use credits to pay TiDB cluster fees before the credits become expired. One credit is equivalent to one U.S. dollar.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-billing.md:119]:
```suggestion
```

File: tidb-cloud/tidb-cloud-billing.md [hunk 4]
```diff
@@ -112,13 +112,24 @@ To view the billing details, perform the following steps:
 2. Click **Billing**.
 3. Click **Bills**. The billing details page is displayed.
 
-## Trial points
+## Credits
 
-During the [PoC](/tidb-cloud/tidb-cloud-poc.md) period, you can use trial points to pay the TiDB cluster fees. One point is equivalent to one U.S. dollar. All your TiDB clusters will be automatically terminated when the trial points are used up.
+TiDB Cloud offers a certain number of credits for Proof of Concept (PoC) users. One credit is equivalent to one U.S. dollar. You can use credits to pay TiDB cluster fees before the credits become expired.
+
```
Comment (TomShawn) [inline tidb-cloud/tidb-cloud-billing.md:118]:
```suggestion

> **Tip:**
>
> To apply for a PoC, see [Perform a Proof of Concept (PoC) with TiDB Cloud](/tidb-cloud/tidb-cloud-poc.md)

```

File: tidb-cloud/tidb-cloud-billing.md [hunk 5]
```diff
@@ -112,13 +112,24 @@ To view the billing details, perform the following steps:
...
+Different types of credits can be used to pay different fees.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-billing.md:123]: Currently, only the "General" type is supported.

Comment (TomShawn) [reply tidb-cloud/tidb-cloud-billing.md:123]: @yuezhanglang PTAL

Comment (TomShawn) [reply tidb-cloud/tidb-cloud-billing.md]: I have deleted this sentence.

File: tidb-cloud/tidb-cloud-billing.md [hunk 6]
```diff
@@ -112,13 +112,33 @@ To view the billing details, perform the following steps:
...
+## Credits
 
-During the [PoC](/tidb-cloud/tidb-cloud-poc.md) period, you can use trial points to pay the TiDB cluster fees. One point is equivalent to one U.S. dollar. All your TiDB clusters will be automatically terminated when the trial points are used up.
+TiDB Cloud offers a certain number of credits for Proof of Concept (PoC) users. One credit is equivalent to one U.S. dollar. You can use credits to pay TiDB cluster fees before the credits become expired.
+
+> **Tip:**
+>
+> To apply for a PoC, see [Perform a Proof of Concept (PoC) with TiDB Cloud](/tidb-cloud/tidb-cloud-poc.md)
+
+The detailed information of your credits is available on the **Credits** page, including your total credits, available credits, current usage, the status, and the expiry date.
```
Comment (TomShawn) [inline tidb-cloud/tidb-cloud-billing.md:123]:
```suggestion
The detailed information of your credits is available on the **Credits** page, including your total credits, available credits, current usage, and the status.
```

**Review by yuezhanglang — APPROVED** [review]
LGTM

File: tidb-cloud/tidb-cloud-billing.md [hunk 7]
```diff
@@ -112,13 +112,33 @@ To view the billing details, perform the following steps:
...
 
-## Trial points
+## Credits
 
-During the [PoC](/tidb-cloud/tidb-cloud-poc.md) period, you can use trial points to pay the TiDB cluster fees. One point is equivalent to one U.S. dollar. All your TiDB clusters will be automatically terminated when the trial points are used up.
+TiDB Cloud offers a certain number of credits for Proof of Concept (PoC) users. One credit is equivalent to one U.S. dollar. You can use credits to pay TiDB cluster fees before the credits become expired.
+
+> **Tip:**
+>
+> To apply for a PoC, see [Perform a Proof of Concept (PoC) with TiDB Cloud](/tidb-cloud/tidb-cloud-poc.md)
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-billing.md:121]:
```suggestion
> To apply for a PoC, see [Perform a Proof of Concept (PoC) with TiDB Cloud](/tidb-cloud/tidb-cloud-poc.md).
```

File: tidb-cloud/tidb-cloud-billing.md [hunk 8]
```diff
@@ -112,13 +112,33 @@ To view the billing details, perform the following steps:
...
-> Once you set up the payment method successfully, the unused trial points will become invalid. The cluster fees that have been deducted from trial points will not be included in your new bill.
+> - After you set up your payment method, the cluster fees are first deducted from your unused credits, then from your payment method.
+> - Credits cannot be used to pay the support plan fees.
+
+> **Warning:**
+>
+> During a PoC process:
+>
+> - if all your credits become expired before you add a payment method, you cannot creat a new cluster. After 3 days, all your created clusters will be recycled. After 7 days, all your backups will be recycled. To resume the process, you can add a payment method.
+> - if all your credits become expired after you add a payment method, your PoC process goes on, and fees are deducted from your payment method.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-billing.md:141]:
```suggestion
> - If all your credits become expired before you add a payment method, you cannot create a new cluster. After 3 days, all your existing clusters will be recycled. After 7 days, all your backups will be recycled. To resume the process, you can add a payment method.
> - If all your credits become expired after you add a payment method, your PoC process goes on, and fees are deducted from your payment method.
```

File: tidb-cloud/release-notes-2022.md [hunk 9]
```diff
@@ -8,6 +8,12 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
 
 This page lists the release notes of [TiDB Cloud](https://en.pingcap.com/tidb-cloud/) in 2022.
 
+## August 16, 2022
+
+* Introduce [Credits](/tidb-cloud/tidb-cloud-billing.md#credits) (previously named as trail points) for PoC users.
+
+    You can now view information about your organization's credits on the **Credits** tab of the **Billing** page, the credits can be used to pay for TiDB Cloud fees. You can [contact us](https://en.pingcap.com/apply-for-poc/) to get credits.
```
Comment (TomShawn) [inline tidb-cloud/release-notes-2022.md:15]:
```suggestion
## August xx, 2022

* Add `2 vCPU, 8 GiB (Beta)` node size of TiDB and TiKV as beta.

    * For each `2 vCPU, 8 GiB (Beta)` TiKV node, the storage size is between 200 GiB and 500 GiB.

    * Suggested usage scenarios:

        * Low-workload production environments for SMB
        * PoC and staging environments
        * Development environments
```

**Review by hnes — APPROVED** [review]

File: tidb-cloud/release-notes-2022.md [hunk 10]
```diff
@@ -8,6 +8,18 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
...
+* Add `2 vCPU, 8 GiB (Beta)` node size of TiDB and TiKV as beta.
+
+    * For each `2 vCPU, 8 GiB (Beta)` TiKV node, the storage size is between 200 GiB and 500 GiB.
+
+    * Suggested usage scenarios:
+
+        * Low-workload production environments for SMB
+        * PoC and staging environments
+        * Development environments
+
```
Comment (TomShawn) [inline tidb-cloud/release-notes-2022.md:22]:
```suggestion

* Introduce [Credits](/tidb-cloud/tidb-cloud-billing.md#credits) (previously named as trail points) for PoC users.

    You can now view information about your organization's credits on the **Credits** tab of the **Billing** page, the credits can be used to pay for TiDB Cloud fees. You can [contact us](https://en.pingcap.com/apply-for-poc/) to get credits.

```

File: tidb-cloud/release-notes-2022.md [hunk 11]
```diff
@@ -8,6 +8,22 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
 
 This page lists the release notes of [TiDB Cloud](https://en.pingcap.com/tidb-cloud/) in 2022.
 
+## August xx, 2022
```
Comment (qiancai) [inline tidb-cloud/release-notes-2022.md:11]:
```suggestion
## August 16, 2022
```

**Review by qiancai — APPROVED** [review]

**Pull request merged**